docs/csharp/programming-guide/concepts/object-oriented-programming.md

```diff
@@ -23,7 +24,7 @@ class SampleClass
 }
 ```
 
-C# also provides types called *structures* that are useful when you don't need support for inheritance or polymorphism.
+C# also provides types called *structures* that are useful when you don't need support for inheritance or polymorphism. For more information, see [Choosing between class and struct](../../../standard/design-guidelines/choosing-between-class-and-struct.md).
```

Comment:
Choosing between struct and classes is more deep than just whether the inheritance and polymorphism are needed or not. So I included the article that explains that in details.